packages/bundle-source/src/index.js

```diff
@@ -407,6 +416,7 @@ ${sourceBundle[entrypoint]}
 return module.exports;
 }
 ${sourceMap}`;
+    return /** @type {const} */ (harden({ moduleFormat, source, sourceMap }));
```

Comment:
Not enough is known for these to be literals. What difference does the const make? (I suspect none.)

Reply:
`moduleFormat` is a union of constant strings. I don’t know the effect of capturing the rest.

Reply:
Yeah, the `const` was needed for an intermediate refactor, but I forgot to remove it after.